containers/ddev-webserver/Dockerfile

```diff
@@ -125,14 +127,14 @@ RUN chmod -R ugo+w /usr/sbin /usr/bin /etc/nginx /var/cache/nginx /var/lib/nginx
 
 RUN source /tmp/ddev/vars && curl -sSL https://github.com/drud/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-${MKCERT_ARCH} -o /usr/local/bin/mkcert && chmod 777 /usr/local/bin/mkcert
 
+RUN git clone https://github.com/longxinH/xhprof.git /var/www/xhprof
```

Comment:
Why you're using git? Maybe better to use pecl source, because no reason to fetch all history